features/font-stretch.yml.dist

```diff
@@ -2,9 +2,7 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: high
-  baseline_low_date: 2017-09-19
-  baseline_high_date: 2020-03-19
+  baseline: false
```

Comment:
This Baseline feature regression is expected. This feature is now discouraged, and points to `font-width` as the alternative. See #2707.